tests/utils.test.js

```diff
@@ -0,0 +1,45 @@
+const { convertToJson } = require("../scripts/utils");
+const {jsonString,yamlString,jsonObject,invalidString} = require("./fixtures/utilsData")
+
+describe('convertToJson', () => {
+  test('should return JSON object if input is valid JSON string', () => {
+    let result;
+    try {
+      result = convertToJson(jsonString);
+    } catch (error) {
+      result = error;
+    }
```

Comment:
Don't assign the `error` to `result` variable. You can directly console.error the error. If the function you are testing fails to return a valid json object if the input is a valid json string, the test would throw an error, and that's what tests are made for.